libs/platform/src/lib/components/form/form-field.ts

```diff
@@ -35,6 +35,10 @@ export abstract class FormField {
      * Indicates in which FormZone a form field should be rendered
      */
     zone: FormZone;
+    /**
+     * Define form field column belongs
+     */
```

Comment:
Do we use still formZone? I though we have columns now

Reply:
Removed